roles/splunk/tasks/configure_auditd.yml

```diff
@@ -0,0 +1,45 @@
+---
+- name: "get {{ splunk_nix_user }} user uid"
+  getent:
+    database: passwd
+    key: "{{ splunk_nix_user }}"
+
+# - debug:
+#     msg: "{{ splunk_nix_user }} user id = {{ ansible_facts.getent_passwd[splunk_nix_user][1] }}"
+
+- name: auditd - set 20-splunk.rules
+  become: true
+  template:
+    src: 20-splunk.rules.j2
+    dest: /etc/audit/rules.d/20-splunk.rules
+    mode: 0600
+    owner: root
+    group: root
+  register: splunk_rule
+
+- name: Get auditd enabled level  # noqa 301 306
```

Comment:
Do we need the `noqa`?

Reply:
The noqa is for ansible-lint. It disables specific checks on a task.  
https://ansible.readthedocs.io/projects/lint/usage/#muting-warnings-to-avoid-false-positives

Since this repo isn't using ansible-lint atm I'll get this removed to avoid any confusion.